content/notes/create-wp-redis-instance.md

```diff
@@ -12,8 +12,12 @@ emoji: 📰
 Here's a step-by-step copy-paste tutorial:
 
 ```bash
+# Setup permissions in case you run PHP-FPM under apache user. If not this step can be skipped.
+chown admin12:apache /home/virtual/benchmark.test/var/www/redis.sock
+chmod 0660 /home/virtual/benchmark.test/var/www/redis.sock
```

Comment:
This is not necessary. We're breaking API and vfs usage and moreover, "apache" is a user within the account's group. It is not a separate group within vfs. See siteXX/fst/etc/passwd in a site's filesystem to get an understanding of this.